http/src/lib.rs

```diff
@@ -407,7 +420,11 @@ fn serve<M: jsonrpc::Metadata, S: jsonrpc::Middleware<M>>(
 		bind_result.and_then(move |(listener, local_addr)| {
 			let allowed_hosts = server_utils::hosts::update(allowed_hosts, &local_addr);
 
-			let http = server::Http::new();
+			let http = {
```

Comment:
[hyper::server::Http::keep_alive](https://docs.rs/hyper/0.11.1/hyper/server/struct.Http.html#method.keep_alive), result is ```&mut Self``` not ```Self```.

```rust
fn keep_alive(&mut self, val: bool) -> &mut Self
```

Because I want ```Self```, I create ```let mut Self```.